airflow/models/connection.py

```diff
@@ -23,6 +23,7 @@
 from json import JSONDecodeError
 from urllib.parse import parse_qsl, quote, unquote, urlencode, urlsplit
 
+import re2
```

Comment:
Can we use `re2` in providers yet?

Reply:
Not sure, I only added this because static linter failed because it was mandating the use of `re2` instead of `re`.

Reply:
Oh this is not actually in providers, I don’t know why I thought so. This is correct.